### ISSUE TITLE
variables in browser.json files

### ISSUE THREAD

**Issue by bhoomisatasiya** [lasso-js/lasso#110]
Hi,
We should be able to pass the config variables to the optimizer .json.
Use case: I have a js widget url , which I store in config and load it on the client side,
I want to pass it to the browser and aggregate it with the page js
Thanks, 
Bhoomika

**Comment by patrick-steele-idem** [comment]
@bhoomisatasiya I agree that this would be helpful. do you have any thoughts how these variables would be passed as input to Lasso.js and how those variables could be referenced in `browser.json` files?

Also, we would need to figure out if/how variables would impact the cache key. It's easy to generate a cache key from a variable with a string value, but what if a variable is a complex type?
